helm_deploy/hmpps-welcome-people-into-prison-ui/Chart.yaml

```diff
@@ -5,7 +5,7 @@ name: hmpps-welcome-people-into-prison-ui
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: 2.1.0
+    version: 2.6.5
```

Comment:
Helm diff shows that the only thing this will change is adding the pod anti-affinity